  * debian/patches/macintosh-gb.diff: New patch to fix <LSGT> and <TLDE> keys.
    Closes: #379148 Thanks Roger Leigh

Not submitted yet.

Index: xkb-data/symbols/macintosh_vndr/gb
===================================================================
--- xkb-data.orig/symbols/macintosh_vndr/gb
+++ xkb-data/symbols/macintosh_vndr/gb
@@ -6,11 +6,12 @@
     // Describes the differences between a very simple en_US
     // keyboard and a very simple U.K. keyboard layout
 
-
     name[Group1]= "Great Britain";
 
     key <AE03> {	[               3,        sterling     	],
 			[      numbersign			]	};
+    key <LSGT> {	[       quoteleft,      asciitilde ] };
+    key <TLDE> {	[         section,       plusminus ] };
 
     // End alphanumeric section
     
